code-challenges.js

```diff
@@ -9,40 +9,88 @@
+
+// var temp = 35
 // var temp = 350
 // var temp = 212
 
+// var temp = 35
+// if (temp < 212) {
+//   console.log("temp is below boiling point");
+// } else if (temp > 212) {
+//   console.log("temp is above boiling point");
+// } else if (temp = 212) {
+//   console.log("temp is at boiling point");
+// }
+
 
 
 // --------------------2) Create the code that will combine two arrays and return the length using the variables provided below. Expected output: 10
 
-var myNumbers1 = [3, 7, 0, 36, -9]
-var myNumbers2 = [8, -7, 22, 9, 13]
+//I will use the .concat() accessor in order to combine the two arrays
+//I will create a variable that holds the value of both given variables combined
+//I will verify that the arrays combined in terminal output
+//I will create a console log for the new variable comboArrays with a length property attached to it
+//Verified output is 10
 
+//Notes on process - I used the syllabus to read through the concat accessor function as I knew that I needed to combine the two provided variables. Following that, I knew that I needed to find the length of the newly created combined variable, which I also found in the syllabus.
+
+// var myNumbers1 = [3, 7, 0, 36, -9]
+// var myNumbers2 = [8, -7, 22, 9, 13]
+// var comboArrays = myNumbers1.concat(myNumbers2);
+// console.log(comboArrays.length);
 
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variables provided below. Expected output: "eilrahc", "atled"
 
-var myString1 = "charlie"
-var myString2 = "delta"
+//I will combine the two variables with a concat accessor to create a new variable
+//I will use the reverse mutator on the new variable to get the desired output
+
+//Revisit this. Cannot figure out why reverse mutator does not work. I am assuming that reverse only works on arrays, but I was unable to successfully establish the given variables as an array.
+
+// var myString1 = "charlie"
+// var myString2 = "delta"
+//
+// console.log(myString1.reverse());
+
 
 
 
 // --------------------4) Create a statement that evaluates two numbers and subtracts the smaller number from the larger number using the two sets of test variables provided below. Expected output: 15, 12
 
-var number1 = 42
-var number2 = 27
+//I will establish a console.log() that allows includes a difference calculation
+
+//Notes on process - I knew that I had to find a way to always subtract that smallest number from the largest number regardless of what value each variable held.
+//I found the Math.max and Math.min at https://www.codegrepper.com/code-examples/javascript/get+the+smaller+number+js and then incorporated that into using the provided variables for a difference calculation.
+
+// var number1 = 42
+// var number2 = 27
 
 // var number1 = 7
 // var number2 = 19
+// console.log(Math.max(number1, number2) - Math.min(number1, number2));
```

Comment:
High level research solutio. I can appreciate that

Also an if, else situation would work here.